PADx_scraper.py

```diff
@@ -75,12 +114,12 @@ def main():
     file = input("Destination file: ")
 
     # Extract and format HTML from URL
-    rem_src = requests.get(url)
-    rem_plain_txt = rem_src.text
-    rem_soup = BeautifulSoup(rem_plain_txt, features="html.parser")
-    print("Scrapping...")
-
-    scrape(rem_soup, file)
+    
+    print("Scraping...")
+    start = time.perf_counter()
+    scrape(url, file)
+    end = time.perf_counter()
+    print(f"Scraping completed in {end - start} seconds.")
```

Comment:
Love the duration timer! One nitpick: print only 2 or 3 decimal places, the rest are negligible and waste screen space

Reply:
Yup. Good point.